README.md

```diff
@@ -1 +1,60 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This project contains index.html file with a header and style.css file to style that header.
+> The project demonestrat the idea of adding file to .gitignore which make the files stay locally and  not push to github reposotary.
+
+
+## Built With
+
+- HTML&CSS
+- VScode 
+
+## Live Demo (if available)
+
+![Capture](https://user-images.githubusercontent.com/5276238/153204533-cad1759d-36a4-422a-8017-d8f15bb0a1a8.PNG)
+
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```

Comment:
- [ ] Kindly modify the `Getting started` section and add some steps on how the users can use your project locally.

Reply:
done 👍